source/javascripts/refills/tabs.js

```diff
@@ -0,0 +1,95 @@
+$(function() {
+  var containerSelector = '[data-tab-wrapper]';
+  var tabListSelector = '[data-tablist]';
+  var tabListItemSelector = '[data-tablist] > li';
```

Comment:
That seems like a lot of duplicated markup. If you add a new item and forget the data attribute it would mess up things.